src/components/Tool/FixPhoneNumbers/Contact.tsx

```diff
@@ -483,7 +482,11 @@ const Contact: React.FC<Props> = ({
                                       setFieldValue('newPhone', e.target.value);
                                       handleSubmit();
                                     }}
-                                    disabled={phoneNumber.source !== appName}
+                                    disabled={
+                                      editableSources.indexOf(
+                                        phoneNumber.source,
+                                      ) === -1
```

Comment:
This should be made into a fucntion as it's used in mutiple places here and the EditContactAddressModal